pkg/awsutils/awsutils.go

```diff
@@ -219,6 +230,7 @@ type DescribeAllENIsResult struct {
 	TagMap      map[string]TagMap
 	TrunkENI    string
 	EFAENIs     map[string]bool
+	MultiCardENIIDs []string
```

Comment:
To continue on the above comment - Maybe we can just check if the ENI is part of `MultiCardENIIDs` as well in `IsUnmanagedENI`. Same goes to `SetUnmanagedENIs`, we can append `MultiCardENIIDs`. Any issues do you see with that approach?

Reply:
Yeah kept it separate for readability. If we have a single DB then we can use the same routines but yes I wanted to keep it separate so that it will be easy to remove in the future release.